modules/permutiveRtdProvider.js

```diff
@@ -236,11 +239,16 @@ export function getSegments (maxSegs) {
     ac: [..._pcrprs, ..._ppam, ...legacySegs],
     rubicon: readSegments('_prubicons'),
     appnexus: readSegments('_papns'),
-    gam: readSegments('_pdfps')
+    gam: readSegments('_pdfps'),
+    ix: legacySegs
```

Comment:
(not introduced in this PR) @dreischer why did we called these segments `legacySegs`? Would `standardCohorts` not be a more indicative name?